examples/templates/suite_proxy/test_async_proxy.py

```diff
@@ -0,0 +1,70 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Proxy auxiliary usage example
+*****************************
+
+:module: test_proxy
+
+:synopsis: show how to use a proxy auxiliary
```

Comment:
```suggestion
:synopsis: show how to use a asynchronus proxy auxiliary
```

Reply:
Please explain further what the meaning of this test should be. What are the Keypoints you want to show ?